### PR TITLE
Progress bar in CLI

### PR DESCRIPTION
**Pull request by slashben**
## Overview
Implemented a progress bar when running KS as a CLI tool. The progress bar starts when the execution gets to control processing and progresses upon completion of each control. This is an important usability improvement for users who scan big clusters and also helps the user to understand if there are outstanding controls that takes more time.

In the same PR I am adding a limitation of control names in the results table (pretty print). The current solution is that the whole control name is printed and it makes the table a mess.

## How to Test

Scan a relatively big cluster (to see the effect)

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: core/pkg/resultshandling/printer/v2/controltable.go [hunk 1]
```diff
@@ -35,7 +35,11 @@ func generateRow(controlSummary reportsummary.IControlSummary, infoToPrintInfo [
 	}
 
 	row[columnSeverity] = getSeverityColumn(controlSummary)
-	row[columnName] = controlSummary.GetName()
+	if len(controlSummary.GetName()) > 50 {
```
Comment (matthyx) [inline core/pkg/resultshandling/printer/v2/controltable.go:38]: I think @fredbi was also implementing something re pretty printing

Comment (slashben) [reply core/pkg/resultshandling/printer/v2/controltable.go:38]: This is not a great solution, in generate we'd need to take into account the terminal with and etc.
Maybe we should look at an existing table implementation as well

Comment (matthyx) [reply core/pkg/resultshandling/printer/v2/controltable.go:38]: https://github.com/kubescape/kubescape/issues/933

**Comment by matthyx** [comment]
so nice, I like it !
now it shows even more that we suck at compiling results...

**Review by matthyx — APPROVED** [review]

**Comment by matthyx** [comment]
thanks @slashben !

**Pull request merged**